### ISSUE TITLE
Bug 54111527: [React Native Gallery -> Home]: Unable to access 'Get started/React Native/Windows Design/GitHub...' etc links using voice access click and show numbers command.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#504]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: Voice Access.

Pre-Requisite: 
Turn on Voice Access (Setting >> Accessibility >> Speech >> Voice Access). 

Repro-Steps:
Launch the WinUI 3 Gallery app.
Navigate to ‘Home’ button present in left navigation pane and invoke it.
Now try to access 'Get started/React Native/Windows Design/GitHub...' etc links using voice access click command and show numbers command.
Actual Result:
Unable to access 'Get started/React Native/Windows Design/GitHub...' etc links using voice access click and show numbers command. When user gives the command “Click Get Started”, no action is performed. So, there is no accessible way to access the 'Controls provide audible feedback' using voice access.
And no number is defined for the links while saying show numbers command.

Expected Result:
User should be able to access 'Get started/React Native/Windows Design/GitHub...' etc links using voice access. When user gives the command “Click Get started”, some action should be performed.

User Impact:
Speech input users who rely on voice access will not be able to access the links present in page.

Link to ADO bug: https://dev.azure.com/microsoft/OS/_workitems/edit/54111527

**Comment by chrisglein** [comment]
Not reproing on version in store.
